src/bci_build/package/gcc/README.md.j2

```diff
@@ -0,0 +1,95 @@
+# The GNU Compiler Collection Container Image
+
+The GNU Compiler Collection (GCC) is an optimizing compiler for various
+architectures and operating systems. It is the default compiler of the GNU
+project and the default compiler for most Linux Distributions including SUSE
+Linux Enterprise and openSUSE.
+
+
+## How to use this image
+
+### Compile your application in a `Dockerfile`
+
+Generally you'd want to compile your application and distribute it as part of a
+container image of your own. For that, create a `Dockerfile` as follows which
+builds a container image based on this image, copies the sources to a working
+directory and compiles the application:
+{%- set reference_no_release = (image.reference | replace("-%RELEASE%", "") ) %}
+
+```Dockerfile
+FROM {{ reference_no_release }}
+WORKDIR /src/
+COPY . /src/
+RUN gcc main.c src1.c src2.c
+CMD ["./a.out"]
+```
+
+It is also possible to compile a static binary with gcc as part of a multistage
+build:
+
+```Dockerfile
+FROM {{ reference_no_release }} as builder
+WORKDIR /src/
+COPY . /src/
+RUN gcc -static main.c src1.c src2.c
+
+FROM registry.suse.com/bci/bci-micro:latest
+WORKDIR /build/
+COPY --from=builder /src/a.out /build/app
+CMD ["/build/app"]
+```
+
+Note that you must build a static binary to deploy it into bci-micro as
+otherwise shared libraries might be missing. You cannot deploy such an app into
+a `scratch` image as it is not possible to statically link glibc.
```

Comment:
```suggestion
Note that you must build a static binary to deploy it into bci-micro;
otherwise shared libraries might be missing. You cannot deploy such an app into
a `scratch` image, as it is not possible to statically link glibc.
```